src/Common/Session/SessionUtil.php

```diff
@@ -82,6 +82,20 @@ public static function coreSessionStart($web_root, $read_only = true): void
         }
     }
 
+    public static function setSession($web_root, $session_key, $session_value): void
```

Comment:
nice!

time to bow down to the Session Master!
![tenor (25)](https://user-images.githubusercontent.com/278968/84224496-89f90680-aa91-11ea-92b3-1db8a6669d97.gif)


What are your thoughts on making the second variable an array to allow setting more than one variable at a time without needing to grab session again (or could also just create another function called setSessions I suppose to support that). So, then call would be something like `OpenEMR\Common\Session\SessionUtil::setSession($web_root, ['pid' => $new_pid_int]);`

Reply:
@sjpadgett , You even brought travis back!

travis wanted to pass the following message:

![tenor (26)](https://user-images.githubusercontent.com/278968/84226680-4f926800-aa97-11ea-8e6b-ecdbbd8d66cb.gif)

Reply:
I was going to do that but wanted to test concept. Just make second param a mixed and test for array in method.

Reply:
omg 😍